analysis/syntactic/structinit/structinit.go

```diff
@@ -230,11 +230,12 @@ func initInfos(
 	allocs []alloced,
 	specs []config.StructInitSpec) (map[*types.Named]InitInfo, error) {
 	infos := make(map[*types.Named]InitInfo)
-	initialized := make(map[config.CodeIdentifier]bool)
+	initialized := make(map[structId]bool)
```

Comment:
Side note: I think it's neat to use code identifiers wherever we need a flexible way to identify code. I think code identifiers should support clearly identifying a struct later on.